ptf_qe_analysis.cpp

```diff
@@ -2,19 +2,134 @@
 #include "ScanPoint.hpp"
 #include "Utilities.hpp"
 #include "FindCircle.hpp"
+#include "TDirectory.h"
```

Comment:
What are all the changes here? Should include such changes in a different PR, or include in the title/description of this one.

Reply:
Probably a test, no related to anything, it is definately not a part of the accelerometer script.